IPython/utils/_process_cli.py

```diff
@@ -12,23 +12,21 @@
 """
 
 # Import cli libraries:
-import clr
```

Comment:
I think this might be a special thing that needs to be imported first for IronPython to work. Best to leave this module alone, I think.